.env.example

```diff
@@ -14,6 +14,10 @@ THINK_CHINATOWN_APPLICATION_ID=<>
 THINK_CHINATOWN_ACCESS_TOKEN=<>
 THINK_CHINATOWN_WEBHOOK_SIGNATURE_KEY=<>
 
+# SEED_DIST_EMAIL is for local development
+# and for attaching your email to the seeded Distributor
+SEED_DIST_EMAIL=
```

Comment:
you might have to add this to the .github workflow if this needs to be defined for tests

Reply:
Fixed, also re failing tests: they're from the webhooks/ charges tests since I'm merging from my forked branch.